### PR TITLE
[Kobo, Mk. 7] Enable the power LED when charging

### PR DESCRIPTION
**Pull request by NiLuJe**
What it says on the tin ;).

Re #6527 

c.f., #5479 for more details about why it's not enabled anywhere else.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6810)

### CODE REVIEW

File: frontend/device/kobo/device.lua [hunk 1]
```diff
@@ -740,6 +750,62 @@ function Kobo:toggleGSensor(toggle)
...
+        return false
+    end
+
+    -- c.f., strace -fittvyy -e trace=ioctl,file,signal,ipc,desc -s 256 -o /tmp/nickel.log -p $(pidof -s nickel) &
+    -- This was observed on a Forma, so I'm mildly hopeful that it's safe on other Mk. 7 devices ;).
+    if toggle == true then
+        -- NOTE: Technically, Nickel forces a toggle off before that, too.
+        --       But since we do that on startup, it shouldn't be necessary here...
+        f:write("ch 4")
+        f:flush()
```
Comment (pazos) [inline frontend/device/kobo/device.lua:780]: Do you need to flush after each write?, if so, wouldn't be more legible to write `f:write("ch4"):flush()`

**Review by poire-z — APPROVED** [review]

File: frontend/device/kobo/device.lua (hunk 1 above)
Comment (NiLuJe) [reply frontend/device/kobo/device.lua:780]: They are individual commands, so, yeah, I assume they really need to be each their own separate write syscall.

(That's certainly how it happens in Nickel).

Comment (pazos) [reply frontend/device/kobo/device.lua:780]: > That doesn't actually work ;p. (io.write doesn't actually return the I/O object)

Duh, sorry for suggesting without testing :p

Comment (NiLuJe) [reply frontend/device/kobo/device.lua:780]: Eh, seemed sensible enough ;).

**Pull request merged**